pkg/validation/test_test.go

```diff
@@ -491,6 +491,8 @@ func TestValidateTestSteps(t *testing.T) {
 	myReference := "my-reference"
 	asReference := "as"
 	yes := true
+	defaultDuration := &prowv1.Duration{Duration: 1 * time.Minute}
+	trueRef := &[]bool{true}[0]
```

Comment:
this is amazing - we usually do

```
yes := true 

// ...
{
  something: &yes
}
```

Reply:
Lol, yeah I'll just do that since the yes bool is already there. I initially had it inlined and then just made it a variable.